lib/ex_statsd.ex

```diff
@@ -271,6 +289,20 @@ defmodule ExStatsD do
     ] |> IO.iodata_to_binary
   end
 
+  defp packet({:event, title, text, opts}, _namespace, tags, _sample_rate) do
+    [
+      "_e",
+      "{#{title |> String.length},#{text |> String.length}}",
+      ":#{title}|#{text}",
+      opts[:priority]         && "|p:#{opts[:priority]}" || "",
+      opts[:alert_type]       && "|t:#{opts[:alert_type]}" || "",
+      opts[:source_type]      && "|s:#{opts[:source_type]}" || "",
+      opts[:aggregation_key]  && "|k:#{opts[:aggregation_key]}" || "",
+      opts[:hostname]         && "|h:#{opts[:hostname]}" || "",
+      tags |> tags_suffix
+    ] |> IO.iodata_to_binary
```

Comment:
also it's probably also good to check the payload size like in ruby gem for example https://github.com/DataDog/dogstatsd-ruby/blob/master/lib/statsd.rb#L297:L318